sharding-proxy/sharding-proxy-transport/sharding-proxy-transport-mysql/src/main/java/org/apache/shardingsphere/shardingproxy/transport/mysql/packet/command/query/MySQLColumnDefinition41Packet.java

```diff
@@ -69,12 +70,17 @@ public MySQLColumnDefinition41Packet(final int sequenceId, final ResultSetMetaDa
                 resultSetMetaData.getColumnLabel(columnIndex), resultSetMetaData.getColumnName(columnIndex), resultSetMetaData.getColumnDisplaySize(columnIndex), 
                 MySQLColumnType.valueOfJDBCType(resultSetMetaData.getColumnType(columnIndex)), resultSetMetaData.getScale(columnIndex));
     }
-    
+
```

Comment:
Please don't change the original indentation.

Reply:
fix